python/servo/devenv_commands.py

```diff
@@ -153,6 +164,16 @@ def grep(self, params):
             ["git"] + ["grep"] + params + ['--'] + grep_paths + [':(exclude)*.min.js'],
             env=self.build_env())
 
+    @Command('update',
+             description='Update rust, cargo and cargo dependencies',
+             category='devenv')
+    def update(self):
```

Comment:
This command is meant to fetch dependencies for offline usage, not update things (e.g. what if you're bisecting in some old history?), so please rename this to `fetch` and update the description.